evalml/tests/conftest.py

```diff
@@ -693,3 +693,36 @@ def _make_data_type(data_type, data):
 @pytest.fixture
 def fraud_100():
     return load_fraud(n_rows=100)
+
+
+@pytest.fixture
+def mock_imbalanced_data_X_y():
+    """Helper function to return an imbalanced binary or multiclass dataset"""
+    def _imbalanced_data_X_y(problem_type, categorical_columns, size):
+        """Take in args to determine the generated dataset to output. For our targets, we maintain a 1:5, or 0.2, class ratio of minority:majority.
```

Comment:
Nit-pick: "Take in args to determine the generated dataset to output." --> "Generates a dummy classification dataset with particular amounts of class imbalance and categorical input columns." Right?